src/builtin_types.md

```diff
@@ -58,7 +58,7 @@ enum Option<a> {
 }
 ```
 
-The type of Grain options (i.e. `Some(1)` or `None`). The `a` is the type of the value. This is useful for functions that might return `None` instead of a value.
+The type of Grain options (i.e. `Some(1)` or `None`). The `a` is the type of the value. This is useful for functions that might return "nullable" values.
```

Comment:
```suggestion
The type of Grain options (i.e. `Some(1)` or `None`). The `a` is the type of the value. This is useful for functions that might return a "nullable" values.
```

I also question if we want to use the term `null`? I think that could cause confusion, I don't quite have a better word though either.

Reply:
Will slightly adjust the wording to make it a bit clearer but I think drawing an analog to null in other languages would be useful

Reply:
> Will slightly adjust the wording to make it a bit clearer but I think drawing an analog to null in other languages would be useful

I think it would be best if we do a comparison we show a bad null example from another language and how it's unsafe and then the safety of `options`. Or if we're making the reference to null I think we should really emphasize the type safety of options in general.